releng/org.eclipse.nebula.nebula-parent/pom.xml

```diff
@@ -33,7 +32,7 @@ Contributors:
 		<jacoco-version>0.8.9</jacoco-version>
 		<easymock-version>5.2.0</easymock-version>
 
-		<target-platform>https://download.eclipse.org/releases/milestone</target-platform>
+		<target-platform>https://download.eclipse.org/releases/2025-03</target-platform>
```

Comment:
This also has the disadvantage of not noticing new problems and becoming stale in just a few weeks.  What do you see as the advantage of this versus milestone or latest?

Reply:
> What do you see as the advantage of this versus milestone or latest?

We have a working build. Also Nebula is actually not meant to be using any latest features, so there is no need to update this. As it is a system property one can of course have a build that set it to milestones/latest/.... in an additional build if we want to have such checks.